### ISSUE TITLE
pretrained shape network

### ISSUE THREAD

**Issue by xiyichen** [MoyGcc/vid2avatar#3]
Thank you for releasing the code! I see you mentioned in the supplementary materials that you did some pretraining on the shape network. Can you share this pretrained model? Also, would it still be able to learn the mask well if trained from scratch?

**Comment by MoyGcc** [comment]
Hi, the pre-trained model can be found in `assets/smpl_init.pth`. 

Yes, it's still possible to learn the mask from scratch but you first need to disable [in_shape_loss](https://github.com/MoyGcc/vid2avatar/blob/main/code/lib/model/loss.py#L38), since in that case, the shape network will be initialized as a sphere and this loss doesn't make sense anymore. But sometimes, training from scratch could also fail (according to my experience). A coarse human shape initialization is encouraged in any case though.
